mmcls/__init__.py

```diff
@@ -48,7 +48,7 @@ def digit_version(version_str: str, length: int = 4):
 
 
 mmcv_minimum_version = '1.4.2'
-mmcv_maximum_version = '1.6.0'
+mmcv_maximum_version = '1.7.0'
```

Comment:
Please also update the mmcv version at https://github.com/open-mmlab/mmclassification/blob/master/requirements/mminstall.txt